language/sygus.tex

```diff
@@ -1045,15 +1045,15 @@ \subsection{Asserting Oracle Constraints and Assumptions}
 \begin{itemize}
 \item $\paren{\constraintoraclekwd\mbox{ } N \mbox{ } 
 \paren{\paren{x_1\mbox{ }\sigma_1} \ldots \paren{x_n\mbox{ }\sigma_n}}\mbox{ }
-\paren{\paren{x_{n+1}\mbox{ }\sigma_{n+1}} \ldots \paren{x_{n+m}\mbox{ }\sigma_{n+m}}}  \mbox{ } t}$
+\paren{\paren{x'_{1}\mbox{ }\sigma'_{1}} \ldots \paren{x'_{m}\mbox{ }\sigma'_{m}}}  \mbox{ } t}$
```

Comment:
Sorry, I don't like `'` either, can this just be `y` instead?

Reply:
What about sigma? can I use sigma', or shoud I use a different symbol to indicate sort?

Otherwise, I guess we have to have the indices with n+1...n+m for the sigmas, so we may as well keep it for the x's too

Reply:
Yes, let's use `\tau`.